docs/web3-eth-miner.rst

```diff
@@ -0,0 +1,224 @@
+.. _eth-miner:
+
+.. include:: include_announcement.rst
+
+==============
+web3.eth.miner
+==============
+
+
+The ``web3-eth-miner`` package allows you to remote control the node's mining operation and set various mining specific settings.
+
+
+.. code-block:: javascript
+
+    import {Miner} from 'web3-eth-miner';
+
+    // "Web3.givenProvider" will be set if in an Ethereum supported browser.
+    const miner = new Miner(Web3.givenProvider || 'ws://some.local-or-remote.node:8546', null, options);
+
+
+------------------------------------------------------------------------------
+
+
+.. include:: include_package-core.rst
+
+
+
+------------------------------------------------------------------------------
+
+
+setExtra
+========
+
+.. code-block:: javascript
+
+    web3.eth.miner.setExtra(extraData, [, callback])
+
+This method allows miner to set extra data during mining the block.
+
+----------
+Parameters
+----------
+
+1. ``extraData`` - ``String``: Extra data which is to be set.
+2. ``Function`` - (optional) Optional callback, returns an error object as first parameter and the result as second.
+
+-------
+Returns
+-------
+
+``Promise<boolean>`` - True if successful.
+
+-------
+Example
+-------
+
+.. code-block:: javascript
+
+    web3.eth.miner.setExtra('Hello').then(console.log);
+    > true
+
+------------------------------------------------------------------------------
+
+
+setGasPrice
+===========
+
+.. code-block:: javascript
+
+    web3.eth.miner.setGasPrice(gasPrice, [, callback])
```

Comment:
this should be: ``miner. setGasPrice(gasPrice, [, callback])``.

Could you change that for all examples? :)